mkdocs.yml

```diff
@@ -20,7 +20,6 @@ nav:
       Gateway: gateway.md
       HTTPRoute: httproute.md
   - Guides:
-      - Introduction: guides.md
```

Comment:
It looks like there are links to this in README.md and index.md, those should probably be updated to link to "Getting Started" now.

Reply:
good catch, I fixed both of these pages